ios/FluentUI.Demo/FluentUI.Demo/Demos/DateTimePickerDemoController.swift

```diff
@@ -36,16 +36,81 @@ class DateTimePickerDemoController: DemoController {
         dateLabel.adjustsFontSizeToFitWidth = true
 
         container.addArrangedSubview(dateLabel)
-        container.addArrangedSubview(createButton(title: "Show date picker", action: #selector(presentDatePicker)))
-        container.addArrangedSubview(createButton(title: "Show date time picker", action: #selector(presentDateTimePicker)))
-        container.addArrangedSubview(createButton(title: "Show date range picker (paged)", action: #selector(presentDateRangePicker)))
-        container.addArrangedSubview(createButton(title: "Show date range picker (tabbed)", action: #selector(presentTabbedDateRangePicker)))
-        container.addArrangedSubview(createButton(title: "Show date time range picker", action: #selector(presentDateTimeRangePicker)))
-        container.addArrangedSubview(createButton(title: "Show picker with custom subtitles or tabs", action: #selector(presentCustomSubtitlePicker)))
+        container.addArrangedSubview(createButton(title: "Show date picker", action: { [weak self] _ in
+            guard let strongSelf = self else {
+                return
```

Comment:
nit: extra new line

Reply:
You mean add a new line?
The convention we use is not to have extra lines on the else of a guard statement and no extra lines before the first statement of a closure or after the last statement of it.